### DISCUSSION TITLE
/*! jQuery UI - v1.12.1 - **2021-05-30** - what is the date is for

### DISCUSSION THREAD

**Discussion by LydieMari** [jquery/jquery-ui#2194] [General]
Hi all

My apologies for the silly question I am about to ask. I am so bad at IT and coding that I need your help about the code below and especially the meaning of the date "2021-05-30".

Basically I have to find the published date of an internet page. In the coding (below), I can find only one date which is that one : 2021-05-30. 
I have checked the released date of the All jQuery UI versions, none matches with 2021-05-30. So by any chance, could you please help me to find out what the date 2021-05-30 refers to?

Many thanks for you help, it's much appreciated.

Lydie

```js
/*! jQuery UI - v1.12.1 - 2021-05-30
--
  | * http://jqueryui.com

  | /*! jQuery UI - v1.12.1 - 2021-05-30
  | * http://jqueryui.com
  | * Copyright jQuery Foundation and other contributors; Licensed MIT */
```

**Comment by Krinkle** [comment]
I suspect you might not be using an official release:

* https://blog.jqueryui.com/2016/09/jquery-ui-1-12-1/
* https://code.jquery.com/ui/1.12.1/jquery-ui.js

```js
/*! jQuery UI - v1.12.1 - 2016-09-14
* http://jqueryui.com
* …
* Copyright jQuery Foundation and other contributors; Licensed MIT */
…
```

One possible source might be the custom download builder at https://jqueryui.com/download/, which generates files with today's date. This suggests that you or someone else on your project, generated a custom version of jQuery UI 1.12.1 at 2021-05-30.
